webapp/src/pages/Onboarding.js

```diff
@@ -1,45 +1,72 @@
-
-  const handleFormSubmit = (event) => {
-    event.preventDefault();
-    setIsSubmitDisabled(true);
-    // navigate to tutorial after submittion
-    console.log("Navigate to tutorial");
-  };
   return (
     <div className="main-container">
       <Row>
         <Col>
-          <h2>Member Onboarding</h2>
-          <Form onSubmit={handleFormSubmit}>
-            <Form.Group className="mb-3" controlId="formBasicPhoneNumber">
-              <Form.Label>Name</Form.Label>
-              <Form.Control
-                // name="name"
-                placeholder="Full Name"
+          <h2>Onboarding</h2>
+          <Carousel fade className="onboarding-carousel rounded overflow-hidden">
+            <Carousel.Item interval={4500}>
+              <img
+                className="d-block w-100"
+                src={onboardingScooters}
+                alt="scooters slide"
               />
-            </Form.Group>
-            <Form.Group className="mb-3" controlId="formBasicPhoneNumber">
-              <Form.Label>Full Address</Form.Label>
-              <Form.Control
-                // name="address"
-                placeholder="e.g. 123 Main Street, Durham, North Carolina 27701"
+              <Carousel.Caption className="bg-dark bg-opacity-75 rounded">
+                <h5>Mobility</h5>
+                <p className="px-3">
+                  We provide clean-energy transportation with electric scooters and bikes
+                </p>
+              </Carousel.Caption>
+            </Carousel.Item>
+            <Carousel.Item interval={4500}>
+              <img
+                className="d-block w-100"
+                src={onboardingDiscounts}
+                alt="discounts slide"
               />
-              <Form.Text className="text-muted">
-                This address will be used to verify your eligibility for membership.
-              </Form.Text>
-            </Form.Group>
-            <Button variant="success" type="submit" disabled={isSubmitDisabled}>
-              Continue
-            </Button>
-          </Form>
+              <Carousel.Caption className="bg-dark bg-opacity-75 rounded">
+                <h5>Save Money</h5>
+                <p className="px-3">Big discounts on things that you buy the most</p>
+              </Carousel.Caption>
+            </Carousel.Item>
+            <Carousel.Item interval={4500}>
+              <img className="d-block w-100" src={onboardingFood} alt="food slide" />
+              <Carousel.Caption className="bg-dark bg-opacity-75 rounded">
+                <h5>Food</h5>
+                <p className="px-3">Hungry? Get food directly to your door</p>
+              </Carousel.Caption>
+            </Carousel.Item>
+          </Carousel>
+          <Link
+            to="/onboarding/signup"
+            className="btn btn-success w-100 p-2 my-2 mt-4 rounded-pill"
+          >
+            Continue Signup
+          </Link>
+          <Button
```

Comment:
Move this to nav (add user nav to this page). May as well finally add logout to the nav.

Also, if we are going to repeat an element like this, make it a component.